From cf5fd5561ff6927f146593828866fbb7a435e1f8 Mon Sep 17 00:00:00 2001
From: Guneshwor Singh <guneshwor.o.singh@intel.com>
Date: Thu, 15 Jun 2017 15:08:06 +0530
Subject: [PATCH 0770/2345] ASoC: Intel: boards: Remove SSP1-codec dai link
 from cnl_rt274 machine

Since NHLT does not have SSP1 endpoint, remove it from the dai link
definitions

Change-Id: I7b08f43d21eeff9decb5722e3af4f142f800b3f7
Signed-off-by: Guneshwor Singh <guneshwor.o.singh@intel.com>
Reviewed-by: Prusty, Subhransu S <subhransu.s.prusty@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Reviewed-by: audio_build <audio_build@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/boards/cnl_rt274.c | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/sound/soc/intel/boards/cnl_rt274.c b/sound/soc/intel/boards/cnl_rt274.c
index a43bb53..a97c51d 100644
--- a/sound/soc/intel/boards/cnl_rt274.c
+++ b/sound/soc/intel/boards/cnl_rt274.c
@@ -296,20 +296,8 @@ struct snd_soc_dai_link cnl_rt274_msic_dailink[] = {
 		.init = cnl_rt274_init,
 	},
 	{
-		.name = "SSP1-Codec",
-		.id = 2,
-		.cpu_dai_name = "SSP1 Pin",
-		.codec_name = "snd-soc-dummy",
-		.codec_dai_name = "snd-soc-dummy-dai",
-		.platform_name = pname,
-		.be_hw_params_fixup = cnl_be_fixup,
-		.ignore_suspend = 1,
-		.no_pcm = 1,
-		.dpcm_playback = 1,
-	},
-	{
 		.name = "dmic01",
-		.id = 3,
+		.id = 2,
 		.cpu_dai_name = "DMIC01 Pin",
 		.codec_name = "dmic-codec",
 		.codec_dai_name = "dmic-hifi",
-- 
2.7.4

